src/main/webapp/dashboard.html

```diff
@@ -36,17 +36,37 @@
         $(document).ready(function () {
             $('#picker-no-time').dateTimePicker({ showTime: false, dateFormat: 'DD/MM/YYYY' }, "startDateTime");
             $('#picker').dateTimePicker(null, "endDateTime");
-            $("#sub").click(function (event) {
+            $("#table-submit").click(function (event) {

```

Comment:
You shouldn't name two DOM elements with the same name, since I have another id called table-submit already.
In other words, you should name your ID with different prefix or something else, but not copy it all
And the better way to put script is in other .js file